src/cmd/gen-sops.ts

```diff
@@ -77,6 +77,7 @@ export const genSops = async (): Promise<void> => {
     if (serviceKeyJson) {
       debug.log('Creating gcp-key.json for vscode.')
       writeFileSync(`${env.ENV_DIR}/gcp-key.json`, JSON.stringify(serviceKeyJson))
+      writeFileSync(`${env.ENV_DIR}/.secrets`, `GCLOUD_SERVICE_KEY='${JSON.stringify(serviceKeyJson)}'`, { flag: 'a' })
```

Comment:
So far the `.secrets` file was read-only and provided by a user.
The GCLOUD_SERVICE_KEY could come from `.secrets`
1) why did you decide to overwrite this file?
2) Could you explain what is the meaning of  `{ flag: 'a' }`?

Reply:
`{ flag: 'a' }` is append, and create & write if not exists.

It is created because the .secrets file is read and parsed for `GCLOUD_SERVICE_KEY` throughout the CLI